_data/hipaa-apps.yml

```diff
@@ -0,0 +1,57 @@
+features:
+  - title: Service discovery
+    description: Apps must look up service endpoints via environment variables, or DNS, or specific IP addresses, or a client-library.
+    available: "yes"
+
+  - title: Service mesh
+    description: After apps successfully find a service, they also need to authenticate to it, encrypt the communication channel, balance the load, and route traffic in specific ways.
+    available: "yes"
+
+  - title: Service Resilience
+    description: Apps must handle retries, exponential back-off, jitter, circuit breakers, thread management, async I/O, throttling, error handling, and a ton of other concerns.
+    available: "yes"
+
+  - title: Metrics / observability
+    description: Apps must use encryption by default. Databases, disk volumes, S3 buckets, and machine images must be encrypted. Network connections must be encrypted.
+    available: "yes"
```

Comment:
This description is not for metrics / observability?

Reply:
Updated with placeholder copy in ef2b3917.